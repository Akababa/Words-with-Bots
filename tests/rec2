pm forejudging 3 1 d
pm max 2 2 d
pm dye 3 3 d
pm preconceived 4 4 d
pm oh 3 5 d
pm reestablished 3 6 d
pm ane 3 7 d
pm bladderlike 4 8 d
pm ut 4 9
pm ah 4 11 d
pm zoosperms 4 12 d
pm ow 4 13 d
pm on 3 14 d
pm requalifying 3 15 d
rm 4 1
rm 4 2
rm 4 4
rm 4 6
rm 4 8
rm 4 12
rm 4 15

ra +oxpebzertsaiolk
lm -n=50
pr b