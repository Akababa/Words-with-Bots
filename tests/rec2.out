rack: OXPEBZERTSAIOLK
OXYPHENBUTAZONE @ 4, 1 acr +1635 (hscore:1635)
OXYPHENBUTAZONE @ 4, 1 acr +1635 (hscore:1635)
OXYPHENBUTAZONE @ 4, 1 acr +1635 (hscore:1635)
OXYPHENBUTAZONE @ 4, 1 acr +1635 (hscore:1635)
        OXY @ 4, 1 acr +139 (hscore:139)
        OXY @ 4, 1 acr +139 (hscore:139)
  LATERIZES @ 1, 1 acr +129 (hscore:129)
  LATERIZES @ 1, 1 acr +129 (hscore:129)
   LATERIZE @ 1, 1 acr +120 (hscore:120)
   LATERIZE @ 1, 1 acr +120 (hscore:120)
PREQUALIFYING @ 2,15 dwn +117 (hscore:117)
PREQUALIFYING @ 2,15 dwn +117 (hscore:117)
  POLARIZES @ 2, 5 acr +110 (hscore:110)
  POLARIZES @ 2, 5 acr +110 (hscore:110)
  POLARIZES @ 2, 5 acr +110 (hscore:110)
  POLARIZES @ 2, 5 acr +110 (hscore:110)
    SALTBOX @ 1, 1 acr +108 (hscore:108)
    SALTBOX @ 1, 1 acr +108 (hscore:108)
 BOOKPLATES @ 2, 4 acr +106 (hscore:106)
 BOOKPLATES @ 2, 4 acr +106 (hscore:106)
 BOOKPLATES @ 2, 4 acr +106 (hscore:106)
 BOOKPLATES @ 2, 4 acr +106 (hscore:106)
REQUALIFYING @ 3,15 dwn +105 (hscore:105)
REQUALIFYING @ 3,15 dwn +105 (hscore:105)
        ONE @ 4,13 acr +105 (hscore:105)
        ONE @ 4,13 acr +105 (hscore:105)
   POLARIZE @ 2, 5 acr +102 (hscore:102)
   POLARIZE @ 2, 5 acr +102 (hscore:102)
   POLARIZE @ 2, 5 acr +102 (hscore:102)
   POLARIZE @ 2, 5 acr +102 (hscore:102)
 AZEOTROPES @ 1, 2 acr +102 (hscore:102)
 AZEOTROPES @ 1, 2 acr +102 (hscore:102)
 AZEOTROPES @ 1, 2 acr +102 (hscore:102)
 AZEOTROPES @ 1, 2 acr +102 (hscore:102)
   OBELIZES @ 2, 6 acr +100 (hscore:100)
   OBELIZES @ 2, 6 acr +100 (hscore:100)
   OBELIZES @ 2, 6 acr +100 (hscore:100)
   OBELIZES @ 2, 6 acr +100 (hscore:100)
 ZOOLATRIES @ 2, 4 acr +99 (hscore:99)
 ZOOLATRIES @ 2, 4 acr +99 (hscore:99)
 ZOOLATRIES @ 2, 4 acr +99 (hscore:99)
 ZOOLATRIES @ 2, 4 acr +99 (hscore:99)
  AZEOTROPE @ 1, 2 acr +99 (hscore:99)
  AZEOTROPE @ 1, 2 acr +99 (hscore:99)
  AZEOTROPE @ 1, 2 acr +99 (hscore:99)
  AZEOTROPE @ 1, 2 acr +99 (hscore:99)
  BOOKPLATE @ 2, 4 acr +98 (hscore:98)
  BOOKPLATE @ 2, 4 acr +98 (hscore:98)
  BOOKPLATE @ 2, 4 acr +98 (hscore:98)
  BOOKPLATE @ 2, 4 acr +98 (hscore:98)
    SOAPBOX @ 2, 4 acr +97 (hscore:97)
    SOAPBOX @ 2, 4 acr +97 (hscore:97)
    POSTBOX @ 2, 5 acr +97 (hscore:97)
    POSTBOX @ 2, 5 acr +97 (hscore:97)
   EPAZOTES @ 2, 6 acr +96 (hscore:96)
   EPAZOTES @ 2, 6 acr +96 (hscore:96)
   EPAZOTES @ 2, 6 acr +96 (hscore:96)
   EPAZOTES @ 2, 6 acr +96 (hscore:96)
   POLEAXES @ 2, 5 acr +94 (hscore:94)
   POLEAXES @ 2, 5 acr +94 (hscore:94)
   POLEAXES @ 2, 5 acr +94 (hscore:94)
   POLEAXES @ 2, 5 acr +94 (hscore:94)
    OBELIZE @ 2, 6 acr +92 (hscore:92)
    OBELIZE @ 2, 6 acr +92 (hscore:92)
    OBELIZE @ 2, 6 acr +92 (hscore:92)
    OBELIZE @ 2, 6 acr +92 (hscore:92)
  ZOOLATERS @ 2, 4 acr +91 (hscore:91)
  ZOOLATERS @ 2, 4 acr +91 (hscore:91)
  ZOOLATERS @ 2, 4 acr +91 (hscore:91)
  ZOOLATERS @ 2, 4 acr +91 (hscore:91)
    POLEAXE @ 2, 5 acr +90 (hscore:90)
    POLEAXE @ 2, 5 acr +90 (hscore:90)
    POLEAXE @ 2, 5 acr +90 (hscore:90)
    POLEAXE @ 2, 5 acr +90 (hscore:90)
FOREJUDGING @ 3, 1 dwn +90 (hscore:90)
FOREJUDGING @ 3, 1 dwn +90 (hscore:90)
  BAPTIZERS @ 1, 1 acr +90 (hscore:90)
  BAPTIZERS @ 1, 1 acr +90 (hscore:90)
    EPAZOTE @ 2, 6 acr +88 (hscore:88)
    EPAZOTE @ 2, 6 acr +88 (hscore:88)
    EPAZOTE @ 2, 6 acr +88 (hscore:88)
    EPAZOTE @ 2, 6 acr +88 (hscore:88)
   ZOOLATER @ 2, 4 acr +87 (hscore:87)
   ZOOLATER @ 2, 4 acr +87 (hscore:87)
   ZOOLATER @ 2, 4 acr +87 (hscore:87)
   ZOOLATER @ 2, 4 acr +87 (hscore:87)
     POLEAX @ 2, 5 acr +86 (hscore:86)
     POLEAX @ 2, 5 acr +86 (hscore:86)
     POLEAX @ 2, 5 acr +86 (hscore:86)
     POLEAX @ 2, 5 acr +86 (hscore:86)
   TAPELIKE @ 1, 1 acr +84 (hscore:84)
   TAPELIKE @ 1, 1 acr +84 (hscore:84)
     EPIZOA @ 2, 6 acr +84 (hscore:84)
     EPIZOA @ 2, 6 acr +84 (hscore:84)
     EPIZOA @ 2, 6 acr +84 (hscore:84)
     EPIZOA @ 2, 6 acr +84 (hscore:84)
   BARKEEPS @ 1, 1 acr +84 (hscore:84)
   BARKEEPS @ 1, 1 acr +84 (hscore:84)
   BOOKLETS @ 2, 4 acr +82 (hscore:82)
   BOOKLETS @ 2, 4 acr +82 (hscore:82)
   BOOKLETS @ 2, 4 acr +82 (hscore:82)
   BOOKLETS @ 2, 4 acr +82 (hscore:82)
   SOAPLIKE @ 2, 4 acr +81 (hscore:81)
   SOAPLIKE @ 2, 4 acr +81 (hscore:81)
   SOAPLIKE @ 2, 4 acr +81 (hscore:81)
   SOAPLIKE @ 2, 4 acr +81 (hscore:81)
    BARKEEP @ 1, 1 acr +81 (hscore:81)
    BARKEEP @ 1, 1 acr +81 (hscore:81)
   BAPTIZES @ 1, 1 acr +81 (hscore:81)
   BAPTIZES @ 1, 1 acr +81 (hscore:81)
   BAPTIZER @ 1, 1 acr +81 (hscore:81)
   BAPTIZER @ 1, 1 acr +81 (hscore:81)
    APELIKE @ 1, 2 acr +81 (hscore:81)
    APELIKE @ 1, 2 acr +81 (hscore:81)
    EPOXIES @ 2, 6 acr +80 (hscore:80)
    EPOXIES @ 2, 6 acr +80 (hscore:80)
    EPOXIES @ 2, 6 acr +80 (hscore:80)
    EPOXIES @ 2, 6 acr +80 (hscore:80)
    PARTOOK @ 1, 1 acr +78 (hscore:78)
    PARTOOK @ 1, 1 acr +78 (hscore:78)
   BOOKREST @ 2, 4 acr +78 (hscore:78)
   BOOKREST @ 2, 4 acr +78 (hscore:78)
   BOOKREST @ 2, 4 acr +78 (hscore:78)
   BOOKREST @ 2, 4 acr +78 (hscore:78)
    BOOKLET @ 2, 4 acr +78 (hscore:78)
    BOOKLET @ 2, 4 acr +78 (hscore:78)
    BOOKLET @ 2, 4 acr +78 (hscore:78)
    BOOKLET @ 2, 4 acr +78 (hscore:78)
    BAPTIZE @ 1, 1 acr +78 (hscore:78)
    BAPTIZE @ 1, 1 acr +78 (hscore:78)
    APELIKE @ 2, 4 acr +78 (hscore:78)
    APELIKE @ 2, 4 acr +78 (hscore:78)
 AEROSOLIZE @ 1, 2 acr +78 (hscore:78)
 AEROSOLIZE @ 1, 2 acr +78 (hscore:78)
 AEROSOLIZE @ 1, 2 acr +78 (hscore:78)
 AEROSOLIZE @ 1, 2 acr +78 (hscore:78)
    POKIEST @ 2, 5 acr +77 (hscore:77)
    POKIEST @ 2, 5 acr +77 (hscore:77)
    POKIEST @ 2, 5 acr +77 (hscore:77)
    POKIEST @ 2, 5 acr +77 (hscore:77)
   OPERABLE @ 2, 6 acr +76 (hscore:76)
   OPERABLE @ 2, 6 acr +76 (hscore:76)
   OPERABLE @ 2, 6 acr +76 (hscore:76)
   OPERABLE @ 2, 6 acr +76 (hscore:76)
   EPIBLAST @ 2, 6 acr +76 (hscore:76)
   EPIBLAST @ 2, 6 acr +76 (hscore:76)
     APEXES @ 2, 6 acr +76 (hscore:76)
     APEXES @ 2, 6 acr +76 (hscore:76)
     POLKAS @ 2, 5 acr +74 (hscore:74)
     POLKAS @ 2, 5 acr +74 (hscore:74)
   POLARIZE @ 2, 7 acr +74 (hscore:74)
   POLARIZE @ 2, 7 acr +74 (hscore:74)
   POLARIZE @ 2, 7 acr +74 (hscore:74)
   POLARIZE @ 2, 7 acr +74 (hscore:74)
    BOOKIES @ 2, 4 acr +74 (hscore:74)
    BOOKIES @ 2, 4 acr +74 (hscore:74)
    BOOKIES @ 2, 4 acr +74 (hscore:74)
    BOOKIES @ 2, 4 acr +74 (hscore:74)
    BOOKERS @ 2, 4 acr +74 (hscore:74)
    BOOKERS @ 2, 4 acr +74 (hscore:74)
    BOOKERS @ 2, 4 acr +74 (hscore:74)
    BOOKERS @ 2, 4 acr +74 (hscore:74)
     POKIES @ 2, 5 acr +73 (hscore:73)
     POKIES @ 2, 5 acr +73 (hscore:73)
     POKIES @ 2, 5 acr +73 (hscore:73)
     POKIES @ 2, 5 acr +73 (hscore:73)
     POKIER @ 2, 5 acr +73 (hscore:73)
     POKIER @ 2, 5 acr +73 (hscore:73)
     POKIER @ 2, 5 acr +73 (hscore:73)
     POKIER @ 2, 5 acr +73 (hscore:73)
     POKERS @ 2, 5 acr +73 (hscore:73)
     POKERS @ 2, 5 acr +73 (hscore:73)
     POKERS @ 2, 5 acr +73 (hscore:73)
     POKERS @ 2, 5 acr +73 (hscore:73)
    PESKIER @ 2, 5 acr +73 (hscore:73)
    PESKIER @ 2, 5 acr +73 (hscore:73)
     PEKOES @ 2, 5 acr +73 (hscore:73)
     PEKOES @ 2, 5 acr +73 (hscore:73)
     PEKOES @ 2, 5 acr +73 (hscore:73)
     PEKOES @ 2, 5 acr +73 (hscore:73)
    OBELISK @ 2, 6 acr +72 (hscore:72)
    OBELISK @ 2, 6 acr +72 (hscore:72)
    OBELISK @ 2, 6 acr +72 (hscore:72)
    OBELISK @ 2, 6 acr +72 (hscore:72)
    APELIKE @ 2, 6 acr +72 (hscore:72)
    APELIKE @ 2, 6 acr +72 (hscore:72)
  SPOLIATOR @ 2, 4 acr +70 (hscore:70)
  SPOLIATOR @ 2, 4 acr +70 (hscore:70)
      ZAXES @ 1, 1 acr +69 (hscore:69)
      ZAXES @ 1, 1 acr +69 (hscore:69)
   SOBERIZE @ 2, 7 acr +69 (hscore:69)
   SOBERIZE @ 2, 7 acr +69 (hscore:69)
   SOBERIZE @ 2, 7 acr +69 (hscore:69)
   SOBERIZE @ 2, 7 acr +69 (hscore:69)
    PATZERS @ 1, 1 acr +69 (hscore:69)
    PATZERS @ 1, 1 acr +69 (hscore:69)
  LAKEPORTS @ 1, 1 acr +69 (hscore:69)
  LAKEPORTS @ 1, 1 acr +69 (hscore:69)
  LAKEPORTS @ 1, 1 acr +69 (hscore:69)
  LAKEPORTS @ 1, 1 acr +69 (hscore:69)
    BAPTIZE @ 2, 8 acr +68 (hscore:68)
    BAPTIZE @ 2, 8 acr +68 (hscore:68)
   SPOLIATE @ 2, 4 acr +66 (hscore:66)
   SPOLIATE @ 2, 4 acr +66 (hscore:66)
   SPOLIATE @ 2, 4 acr +66 (hscore:66)
   SPOLIATE @ 2, 4 acr +66 (hscore:66)
   ROOKIEST @ 2, 4 acr +66 (hscore:66)
   ROOKIEST @ 2, 4 acr +66 (hscore:66)
   ROOKIEST @ 2, 4 acr +66 (hscore:66)
   ROOKIEST @ 2, 4 acr +66 (hscore:66)
   POLESTAR @ 2, 5 acr +66 (hscore:66)
   POLESTAR @ 2, 5 acr +66 (hscore:66)
   POLESTAR @ 2, 5 acr +66 (hscore:66)
   POLESTAR @ 2, 5 acr +66 (hscore:66)
   POLARISE @ 2, 5 acr +66 (hscore:66)
   POLARISE @ 2, 5 acr +66 (hscore:66)
   POLARISE @ 2, 5 acr +66 (hscore:66)
   POLARISE @ 2, 5 acr +66 (hscore:66)
   PELTRIES @ 2, 5 acr +66 (hscore:66)
   PELTRIES @ 2, 5 acr +66 (hscore:66)
   PELORIAS @ 2, 5 acr +66 (hscore:66)
   PELORIAS @ 2, 5 acr +66 (hscore:66)
   PELORIAS @ 2, 5 acr +66 (hscore:66)
   PELORIAS @ 2, 5 acr +66 (hscore:66)
    LOOKERS @ 2, 4 acr +66 (hscore:66)
    LOOKERS @ 2, 4 acr +66 (hscore:66)
    LOOKERS @ 2, 4 acr +66 (hscore:66)
    LOOKERS @ 2, 4 acr +66 (hscore:66)
   LATERIZE @ 2, 7 acr +66 (hscore:66)
   LATERIZE @ 2, 7 acr +66 (hscore:66)
     AXLIKE @ 1, 2 acr +66 (hscore:66)
     AXLIKE @ 1, 2 acr +66 (hscore:66)
   SOLARIZE @ 2, 7 acr +65 (hscore:65)
   SOLARIZE @ 2, 7 acr +65 (hscore:65)
   SOLARIZE @ 2, 7 acr +65 (hscore:65)
   SOLARIZE @ 2, 7 acr +65 (hscore:65)
   POSTORAL @ 2, 5 acr +65 (hscore:65)
   POSTORAL @ 2, 5 acr +65 (hscore:65)
     OKAPIS @ 2, 6 acr +65 (hscore:65)
     OKAPIS @ 2, 6 acr +65 (hscore:65)
   LOOPIEST @ 2, 4 acr +65 (hscore:65)
   LOOPIEST @ 2, 4 acr +65 (hscore:65)
   LOOPIEST @ 2, 4 acr +65 (hscore:65)
   LOOPIEST @ 2, 4 acr +65 (hscore:65)
      TOPOI @ 6, 2 dwn +64 (hscore:64)
      TOPOI @ 6, 2 dwn +64 (hscore:64)
    POSTBOX @ 2, 7 acr +64 (hscore:64)
    POSTBOX @ 2, 7 acr +64 (hscore:64)
   OBSOLETE @ 2, 6 acr +64 (hscore:64)
   OBSOLETE @ 2, 6 acr +64 (hscore:64)
   OBSOLETE @ 2, 6 acr +64 (hscore:64)
   OBSOLETE @ 2, 6 acr +64 (hscore:64)
    OBELIZE @ 2, 8 acr +64 (hscore:64)
    OBELIZE @ 2, 8 acr +64 (hscore:64)
    OBELIZE @ 2, 8 acr +64 (hscore:64)
    OBELIZE @ 2, 8 acr +64 (hscore:64)
   EPISTLER @ 2, 6 acr +64 (hscore:64)
   EPISTLER @ 2, 6 acr +64 (hscore:64)
    TOELIKE @ 2, 4 acr +63 (hscore:63)
    TOELIKE @ 2, 4 acr +63 (hscore:63)
    TOELIKE @ 2, 4 acr +63 (hscore:63)
    TOELIKE @ 2, 4 acr +63 (hscore:63)
   PARTIBLE @ 1, 1 acr +63 (hscore:63)
   PARTIBLE @ 1, 1 acr +63 (hscore:63)
   PARBOILS @ 1, 1 acr +63 (hscore:63)
   PARBOILS @ 1, 1 acr +63 (hscore:63)
    LAZIEST @ 1, 1 acr +63 (hscore:63)
    LAZIEST @ 1, 1 acr +63 (hscore:63)
     KAZOOS @ 1, 1 acr +63 (hscore:63)
     KAZOOS @ 1, 1 acr +63 (hscore:63)
    ASTILBE @ 1, 2 acr +63 (hscore:63)
    ASTILBE @ 1, 2 acr +63 (hscore:63)
    TRAPEZE @ 2, 8 acr +62 (hscore:62)
    TRAPEZE @ 2, 8 acr +62 (hscore:62)
    SPELTER @ 2, 4 acr +62 (hscore:62)
    SPELTER @ 2, 4 acr +62 (hscore:62)
    ROOKIES @ 2, 4 acr +62 (hscore:62)
    ROOKIES @ 2, 4 acr +62 (hscore:62)
    ROOKIES @ 2, 4 acr +62 (hscore:62)
    ROOKIES @ 2, 4 acr +62 (hscore:62)
    POLOIST @ 2, 5 acr +62 (hscore:62)
    POLOIST @ 2, 5 acr +62 (hscore:62)
    POLITER @ 2, 5 acr +62 (hscore:62)
    POLITER @ 2, 5 acr +62 (hscore:62)
    POLITER @ 2, 5 acr +62 (hscore:62)
    POLITER @ 2, 5 acr +62 (hscore:62)
    POETIZE @ 2, 8 acr +62 (hscore:62)
    POETIZE @ 2, 8 acr +62 (hscore:62)
    POETIZE @ 2, 8 acr +62 (hscore:62)
    POETIZE @ 2, 8 acr +62 (hscore:62)
    PELTERS @ 2, 5 acr +62 (hscore:62)
    PELTERS @ 2, 5 acr +62 (hscore:62)
    PELOTAS @ 2, 5 acr +62 (hscore:62)
    PELOTAS @ 2, 5 acr +62 (hscore:62)
    PELOTAS @ 2, 5 acr +62 (hscore:62)
    PELOTAS @ 2, 5 acr +62 (hscore:62)
    PELORIA @ 2, 5 acr +62 (hscore:62)
    PELORIA @ 2, 5 acr +62 (hscore:62)
    PELORIA @ 2, 5 acr +62 (hscore:62)
    PELORIA @ 2, 5 acr +62 (hscore:62)
    PELITES @ 2, 5 acr +62 (hscore:62)
    PELITES @ 2, 5 acr +62 (hscore:62)
    PALTERS @ 2, 5 acr +62 (hscore:62)
    PALTERS @ 2, 5 acr +62 (hscore:62)
    PALIEST @ 2, 5 acr +62 (hscore:62)
    PALIEST @ 2, 5 acr +62 (hscore:62)
   BRITZKAS @ 2, 7 acr +62 (hscore:62)
    SOAPBOX @ 2, 7 acr +61 (hscore:61)
    SOAPBOX @ 2, 7 acr +61 (hscore:61)
      OKAPI @ 2, 6 acr +61 (hscore:61)
      OKAPI @ 2, 6 acr +61 (hscore:61)
    LOOPIER @ 2, 4 acr +61 (hscore:61)
    LOOPIER @ 2, 4 acr +61 (hscore:61)
    LOOPIER @ 2, 4 acr +61 (hscore:61)
    LOOPIER @ 2, 4 acr +61 (hscore:61)
    LOOPERS @ 2, 4 acr +61 (hscore:61)
    LOOPERS @ 2, 4 acr +61 (hscore:61)
    LOOPERS @ 2, 4 acr +61 (hscore:61)
    LOOPERS @ 2, 4 acr +61 (hscore:61)
    LOOBIES @ 2, 4 acr +61 (hscore:61)
    LOOBIES @ 2, 4 acr +61 (hscore:61)
    LOOBIES @ 2, 4 acr +61 (hscore:61)
    LOOBIES @ 2, 4 acr +61 (hscore:61)
   ESPALIER @ 2, 6 acr +61 (hscore:61)
   ESPALIER @ 2, 6 acr +61 (hscore:61)
    APOSTLE @ 2, 4 acr +61 (hscore:61)
    APOSTLE @ 2, 4 acr +61 (hscore:61)
    APOSTLE @ 2, 4 acr +61 (hscore:61)
    APOSTLE @ 2, 4 acr +61 (hscore:61)
    APOSTIL @ 2, 4 acr +61 (hscore:61)
    APOSTIL @ 2, 4 acr +61 (hscore:61)
       SYPH @ 4, 2 acr +60 (hscore:60)
     PATZER @ 1, 1 acr +60 (hscore:60)
     PATZER @ 1, 1 acr +60 (hscore:60)
    PARBOIL @ 1, 1 acr +60 (hscore:60)
    PARBOIL @ 1, 1 acr +60 (hscore:60)
   OPERATES @ 2, 6 acr +60 (hscore:60)
   OPERATES @ 2, 6 acr +60 (hscore:60)
   OPERATES @ 2, 6 acr +60 (hscore:60)
   OPERATES @ 2, 6 acr +60 (hscore:60)
   LAKEPORT @ 1, 1 acr +60 (hscore:60)
   LAKEPORT @ 1, 1 acr +60 (hscore:60)
   LAKEPORT @ 1, 1 acr +60 (hscore:60)
   LAKEPORT @ 1, 1 acr +60 (hscore:60)
      KAZOO @ 1, 1 acr +60 (hscore:60)
      KAZOO @ 1, 1 acr +60 (hscore:60)
      EBOOK @ 2, 6 acr +60 (hscore:60)
      EBOOK @ 2, 6 acr +60 (hscore:60)
      EBOOK @ 2, 6 acr +60 (hscore:60)
      EBOOK @ 2, 6 acr +60 (hscore:60)
     BRAIZE @ 2, 9 acr +60 (hscore:60)
     BRAIZE @ 2, 9 acr +60 (hscore:60)
      BLAZE @ 2,10 acr +60 (hscore:60)
      BLAZE @ 2,10 acr +60 (hscore:60)
     BAZOOS @ 1, 1 acr +60 (hscore:60)
     BAZOOS @ 1, 1 acr +60 (hscore:60)
   BAPTIZES @ 2, 7 acr +60 (hscore:60)
   BAPTIZES @ 2, 7 acr +60 (hscore:60)
   BALKIEST @ 1, 1 acr +60 (hscore:60)
   BALKIEST @ 1, 1 acr +60 (hscore:60)
     BAIZES @ 1, 1 acr +60 (hscore:60)
     BAIZES @ 1, 1 acr +60 (hscore:60)
    AZOTISE @ 1, 2 acr +60 (hscore:60)
    AZOTISE @ 1, 2 acr +60 (hscore:60)
    AZOTISE @ 1, 2 acr +60 (hscore:60)
    AZOTISE @ 1, 2 acr +60 (hscore:60)
     AZOLES @ 1, 2 acr +60 (hscore:60)
     AZOLES @ 1, 2 acr +60 (hscore:60)
     AZOLES @ 1, 2 acr +60 (hscore:60)
     AZOLES @ 1, 2 acr +60 (hscore:60)
     ASLEEP @ 1, 2 acr +60 (hscore:60)
     ASLEEP @ 1, 2 acr +60 (hscore:60)
     APEXES @ 1, 2 acr +60 (hscore:60)
     APEXES @ 1, 2 acr +60 (hscore:60)
      APEEK @ 2, 6 acr +60 (hscore:60)
      APEEK @ 2, 6 acr +60 (hscore:60)
    REALIZE @ 2, 8 acr +58 (hscore:58)
    REALIZE @ 2, 8 acr +58 (hscore:58)
      PRIZE @ 2,10 acr +58 (hscore:58)
      PRIZE @ 2,10 acr +58 (hscore:58)
   PRETZELS @ 2, 7 acr +58 (hscore:58)
   PRETZELS @ 2, 7 acr +58 (hscore:58)
   PREBOOKS @ 2, 7 acr +58 (hscore:58)
   PREBOOKS @ 2, 7 acr +58 (hscore:58)
   PREBOOKS @ 2, 7 acr +58 (hscore:58)
   PREBOOKS @ 2, 7 acr +58 (hscore:58)
     POLITE @ 2, 5 acr +58 (hscore:58)
     POLITE @ 2, 5 acr +58 (hscore:58)
     POLITE @ 2, 5 acr +58 (hscore:58)
     POLITE @ 2, 5 acr +58 (hscore:58)
     POLIOS @ 2, 5 acr +58 (hscore:58)
     POLIOS @ 2, 5 acr +58 (hscore:58)
     POLERS @ 2, 5 acr +58 (hscore:58)
     POLERS @ 2, 5 acr +58 (hscore:58)
     POLERS @ 2, 5 acr +58 (hscore:58)
     POLERS @ 2, 5 acr +58 (hscore:58)
     POLEIS @ 2, 5 acr +58 (hscore:58)
     POLEIS @ 2, 5 acr +58 (hscore:58)
     POLEIS @ 2, 5 acr +58 (hscore:58)
     POLEIS @ 2, 5 acr +58 (hscore:58)
     POLARS @ 2, 5 acr +58 (hscore:58)
     POLARS @ 2, 5 acr +58 (hscore:58)
     PELTER @ 2, 5 acr +58 (hscore:58)
     PELTER @ 2, 5 acr +58 (hscore:58)
     PELOTA @ 2, 5 acr +58 (hscore:58)
     PELOTA @ 2, 5 acr +58 (hscore:58)
     PELOTA @ 2, 5 acr +58 (hscore:58)
     PELOTA @ 2, 5 acr +58 (hscore:58)
     PELITE @ 2, 5 acr +58 (hscore:58)
     PELITE @ 2, 5 acr +58 (hscore:58)
     PALTER @ 2, 5 acr +58 (hscore:58)
     PALTER @ 2, 5 acr +58 (hscore:58)
     PALIER @ 2, 5 acr +58 (hscore:58)
     PALIER @ 2, 5 acr +58 (hscore:58)
     PALETS @ 2, 5 acr +58 (hscore:58)
     PALETS @ 2, 5 acr +58 (hscore:58)
     PALEST @ 2, 5 acr +58 (hscore:58)
     PALEST @ 2, 5 acr +58 (hscore:58)
      KIBLA @ 9,14 dwn +58 (hscore:58)
      BRAZE @ 2,10 acr +58 (hscore:58)
      BRAZE @ 2,10 acr +58 (hscore:58)
    BOXLIKE @ 2, 8 acr +58 (hscore:58)
    BOXLIKE @ 2, 8 acr +58 (hscore:58)
    BOXLIKE @ 2, 8 acr +58 (hscore:58)
    BOXLIKE @ 2, 8 acr +58 (hscore:58)
      BOOZE @ 2,10 acr +58 (hscore:58)
      BOOZE @ 2,10 acr +58 (hscore:58)
      BOOZE @ 2,10 acr +58 (hscore:58)
      BOOZE @ 2,10 acr +58 (hscore:58)
      BAIZE @ 2,10 acr +58 (hscore:58)
      BAIZE @ 2,10 acr +58 (hscore:58)
    SOAPIER @ 2, 4 acr +57 (hscore:57)
    SOAPIER @ 2, 4 acr +57 (hscore:57)
    SOAPIER @ 2, 4 acr +57 (hscore:57)
    SOAPIER @ 2, 4 acr +57 (hscore:57)
  SAPROLITE @ 1, 1 acr +57 (hscore:57)
  SAPROLITE @ 1, 1 acr +57 (hscore:57)
  SAPROLITE @ 1, 1 acr +57 (hscore:57)
  SAPROLITE @ 1, 1 acr +57 (hscore:57)
    SALTBOX @ 2, 7 acr +57 (hscore:57)
    SALTBOX @ 2, 7 acr +57 (hscore:57)
     POSTAL @ 2, 5 acr +57 (hscore:57)
     POSTAL @ 2, 5 acr +57 (hscore:57)
     PESTLE @ 2, 5 acr +57 (hscore:57)
     PESTLE @ 2, 5 acr +57 (hscore:57)
    PESTIER @ 2, 5 acr +57 (hscore:57)
    PESTIER @ 2, 5 acr +57 (hscore:57)
     PASTIL @ 2, 5 acr +57 (hscore:57)
    PASTIER @ 2, 5 acr +57 (hscore:57)
    PASTIER @ 2, 5 acr +57 (hscore:57)
     PASTEL @ 2, 5 acr +57 (hscore:57)
     PASTEL @ 2, 5 acr +57 (hscore:57)
    LATEXES @ 1, 1 acr +57 (hscore:57)
    LATEXES @ 1, 1 acr +57 (hscore:57)
  LABORITES @ 1, 1 acr +57 (hscore:57)
  LABORITES @ 1, 1 acr +57 (hscore:57)
  LABORITES @ 1, 1 acr +57 (hscore:57)
  LABORITES @ 1, 1 acr +57 (hscore:57)
    BRITSKA @ 1,10 dwn +57 (hscore:57)
      BAZOO @ 1, 1 acr +57 (hscore:57)
      BAZOO @ 1, 1 acr +57 (hscore:57)
    BATLIKE @ 1, 1 acr +57 (hscore:57)
    BATLIKE @ 1, 1 acr +57 (hscore:57)
   BARKIEST @ 1, 1 acr +57 (hscore:57)
   BARKIEST @ 1, 1 acr +57 (hscore:57)
    BALKIER @ 1, 1 acr +57 (hscore:57)
    BALKIER @ 1, 1 acr +57 (hscore:57)
    BALKERS @ 1, 1 acr +57 (hscore:57)
    BALKERS @ 1, 1 acr +57 (hscore:57)
   BAKERIES @ 1, 1 acr +57 (hscore:57)
   BAKERIES @ 1, 1 acr +57 (hscore:57)
      BAIZE @ 1, 1 acr +57 (hscore:57)
      BAIZE @ 1, 1 acr +57 (hscore:57)
     AZOTES @ 1, 2 acr +57 (hscore:57)
     AZOTES @ 1, 2 acr +57 (hscore:57)
     AZOTES @ 1, 2 acr +57 (hscore:57)
     AZOTES @ 1, 2 acr +57 (hscore:57)
     SPELTZ @ 2, 4 acr +56 (hscore:56)
     SPELTZ @ 2, 4 acr +56 (hscore:56)
     SLEAZE @ 2, 9 acr +56 (hscore:56)
     SLEAZE @ 2, 9 acr +56 (hscore:56)
    POLEAXE @ 2, 8 acr +56 (hscore:56)
    POLEAXE @ 2, 8 acr +56 (hscore:56)
    POLEAXE @ 2, 8 acr +56 (hscore:56)
    POLEAXE @ 2, 8 acr +56 (hscore:56)
    OBLATES @ 2, 6 acr +56 (hscore:56)
    OBLATES @ 2, 6 acr +56 (hscore:56)
    OBLATES @ 2, 6 acr +56 (hscore:56)
    OBLATES @ 2, 6 acr +56 (hscore:56)
    OBLASTI @ 2, 6 acr +56 (hscore:56)
    OBLASTI @ 2, 6 acr +56 (hscore:56)
    OBELISE @ 2, 6 acr +56 (hscore:56)
    OBELISE @ 2, 6 acr +56 (hscore:56)
    OBELISE @ 2, 6 acr +56 (hscore:56)
    OBELISE @ 2, 6 acr +56 (hscore:56)
    OBELIAS @ 2, 6 acr +56 (hscore:56)
    OBELIAS @ 2, 6 acr +56 (hscore:56)
    OBELIAS @ 2, 6 acr +56 (hscore:56)
    OBELIAS @ 2, 6 acr +56 (hscore:56)
    EROTIZE @ 2, 8 acr +56 (hscore:56)
board:
0123456789012345
1...!..*.*..!...
2.M'..?...?..'..
3FAD.ORA...'..OR
4!.Y*H.N?UTA*ON!
5R.ER.EEL'.HOW.Q
6E?.E.S.A.*.O.?U
7J..C'T.D..'S..A
8U..O.A.D...P..L
9D..N'B.E..'E..I
0G?.C.L.R.*.R.?F
1I.'E.I'L'..M'.Y
2N..I.S.I...S..I
3G'.V'H.K..'..'N
4..'E.E.E.?..'.G
5...D.D*.*..!...
